cmake/CPackConfig.cmake

```diff
@@ -0,0 +1,44 @@
+IF(NOT CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    MESSAGE(FATAL_ERROR "Cannot configure CPack to generate Debian/RPM packages on non-linux systems.")
+ENDIF()
+string(TOLOWER "${CMAKE_PROJECT_NAME}" CPACK_PACKAGE_NAME)
+SET(CPACK_PACKAGE_VERSION_MAJOR ${OSRM_VERSION_MAJOR})
+SET(CPACK_PACKAGE_VERSION_MINOR ${OSRM_VERSION_MINOR})
+SET(CPACK_PACKAGE_VERSION_PATCH ${OSRM_VERSION_PATCH})
+SET(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}")
+
+SET(CPACK_INCLUDE_TOPLEVEL_DIRECTORY "FALSE")
+SET(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_SOURCE_DIR}/README.md")
+SET(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Open Source Routing Machine (OSRM) is a high-performance routing engine. It combines sophisticated routing algorithms with the open and free data of the OpenStreetMap.")
+SET(CPACK_PACKAGE_CONTACT "Dennis Luxen <info@project-osrm.org>")
```

Comment:
I think we should change that. It's two years now that Dennis is no longer involved with OSRM development.